charts/kong/README.md

```diff
@@ -288,6 +294,118 @@ scenario and several settings that are useful when using multiple controllers:
 Standalone controllers require a database-backed Kong instance, as DB-less mode
 requires that a single controller generate a complete Kong configuration.
 
+### Hybrid mode
```

Comment:
Out of scope for this PR but we should consider moving such documentation to it's own markdown page. Just leaving a note. This is probably already on your mind.

Reply:
Yeah--the whole README is quite long at this point, and splitting some of the finer details into separate references is something we should do, just not now. Created #164.